### PR TITLE
Use make env repo for conda

### PR DESCRIPTION
**Pull request by acomodi**

### CODE REVIEW

File: symbiflow.py [hunk 1]
```diff
@@ -505,9 +505,11 @@ def run(self):
 
                 chip = self.family + self.device
 
+                conda_dir = os.getenv("CONDA_PREFIX", None)
```
Comment (litghost) [inline symbiflow.py:508]: In general, don't depend on conda directly.  FYI @HackerFoo

Comment (HackerFoo) [reply symbiflow.py:508]: Yes, code should be independent of the package management system used if possible.

**Comment by litghost** [comment]
```
cd . && symbiflow_write_bitstream -d artix7 -f blinky.fasm -p xc7a35tcsg324-1 -b blinky.bit
Writing bitstream ...
realpath: /tmpfs/src/github/fpga-tool-perf/third_party/prjxray-db/artix7: No such file or directory
```

**Comment by acomodi** [comment]
> ```
> cd . && symbiflow_write_bitstream -d artix7 -f blinky.fasm -p xc7a35tcsg324-1 -b blinky.bit
> Writing bitstream ...
> realpath: /tmpfs/src/github/fpga-tool-perf/third_party/prjxray-db/artix7: No such file or directory
> ```

Yep, to go green this requires a new install package from https://github.com/SymbiFlow/symbiflow-arch-defs/pull/1504

**Pull request merged**